.github/workflows/python_unit_test.yaml

```diff
@@ -1,21 +1,32 @@
 name: Run Python unit tests using Poetry, with a Postgres backend
 
-on: 
+on:
   workflow_call:
     inputs:
       setup-script-location:
         description: 'A URL identifying a Bash script that can run commands needed for service-specific test setup'
-        required: true
+        required: false
         type: string
-    
-env: 
+      python-version:
+        description: 'The Python version to use and defaults to latest'
+        required: false
+        default: 'latest'
+        type: string
+      postgres-needed:
```

Comment:
unless i'm missing something, `postgres-needed` is not used anywhere to disable starting the postgres service if it's not needed

Reply:
nvm, i see you just updated that :-)